src/pages/Software.pug

```diff
@@ -92,7 +92,116 @@ block article
         This means that the encoded data also needs to contain the information of how many bits were padded to the end.
         Due to this additional complexity, other conversions than from byte to tryte are not implemented in our software right now.
 
-        # Decoding Data from Sequenced DNA
+        # Decoding and Error Correction on Sequencing Data
 
     - var imgpath = "" + require("../assets/img/Software/decoding.svg")["default"]
-    +image(2, imgpath, "Overview of the steps to retrieve information from DNA.", 60,0.8,true)
+    +image(2, imgpath, "Overview over the steps for retrieving information from DNA.", 70)
+
+    :markdown-it
+
+        As you can see in figure 2, decoding mainly works just as encoding but backwards.
+        However, one additional step is necessary before data can be decoded.
+        From the sequencing data, we first must retrieve the original DNA sequence that was used for synthesis.
+        To do so, we must understand what happens in synthesis.
+        As discussed before, an unknown number of nucleotides is attached in each step.
+        This is due to the semi-specific approach.
+        In addition to that, various primers and other sequences are added to both ends of the encoded sequence.
+        This partly happends during synthesis and partly in preparation for sequencing.
+
+    - var imgpath = "" + require("../assets/img/Software/fastq_example.jpg")["default"]
+    +image(3, imgpath, "Example of DNA sequences in a fastq file", 100)
+
+    :markdown-it
+
+        The decoding starts with a `fastq` file containing sequencing data.
+        Figure 3 shows an example of such a file.
+        Each read from sequencing is expressed by four lines.
+        While the first and third line can contain an identifier and description of the sequence, the second and fourth line are more important to us.
+        The second line contains the actual DNA sequence.
+        In line four the quality of the sequence is encoded.
+        This is important to assess how confident the sequencing was.
+        The quality is encoded in values reaching from 0x21 (33 in decimal) to 0x7E (126) and displayed as the corresponding ASCII character.
+        So a "!" marks the lowest quality, while a "~" stands for the highest quality.
+
+        ## Removing Primers and Cleaning Up Data
+
+    :markdown-it
+
+        The first step necessary for decoding is the removal of the primer since this does not contain any relevant information.
```

Comment:
```suggestion
        The first step necessary for decoding is the removal of the primer sequence since it does not contain any relevant information.
```